kolibri/plugins/learn/assets/test/views/learn-top-nav.spec.js

```diff
@@ -1,136 +0,0 @@
-import VueRouter from 'vue-router';
-import { mount, createLocalVue } from '@vue/test-utils';
-import LearnTopNav from '../../src/views/LearnTopNav';
-import makeStore from '../makeStore';
-// eslint-disable-next-line import/named
-import useCoreLearn, { useCoreLearnMock } from '../../src/composables/useCoreLearn';
-
-jest.mock('../../src/composables/useCoreLearn');
-
-jest.mock('plugin_data', () => {
-  return {
-    __esModule: true,
-    default: {
-      accessibilityLabels: [],
-      categories: [],
-      gradeLevels: [],
-      languages: [],
-      channels: [],
-      learnerNeeds: [],
-    },
-  };
-});
-
-const localVue = createLocalVue();
-localVue.use(VueRouter);
-
-const router = new VueRouter({
-  routes: [
-    { path: '/home', name: 'HOME' },
-    { path: '/library', name: 'LIBRARY' },
-    { path: '/bookmarks', name: 'BOOKMARKS' },
-  ],
-});
-
-function makeWrapper(options) {
-  return mount(LearnTopNav, {
-    ...options,
-    stubs: {
-      breadcrumbs: true,
-      contentUnavailablePage: true,
-      topicsPage: true,
-      TotalPoints: true,
-    },
-    localVue,
-    router,
-  });
-}
-
-function getElements(wrapper) {
-  return {
-    // hrefs need to match the routes in the mock router above
-    homeLink: () => wrapper.find('[href="#/home"]'),
-    bookmarksLink: () => wrapper.find('[href="#/bookmarks"]'),
-    libraryLink: () => wrapper.find('[href="#/library"]'),
-    tabLinks: () => wrapper.findAllComponents({ name: 'NavbarLink' }),
-  };
-}
-
-describe('learn plugin index page', () => {
-  let store;
-
-  const setSessionUserKind = kind => {
-    store.state.core.session.kind = [kind];
-    store.state.core.session.user_id = 'test';
-  };
-  const setMemberships = memberships => {
-    useCoreLearn.mockImplementation(() =>
-      useCoreLearnMock({ inClasses: Boolean(memberships.length) })
-    );
-  };
-  const setPageName = pageName => {
-    store.state.pageName = pageName;
-  };
-  const setCanAccessUnassignedContent = canAccess => {
-    store.state.canAccessUnassignedContentSetting = canAccess;
-  };
-
-  beforeEach(() => {
-    store = makeStore();
-  });
-
-  it('there are no tabs if showing content unavailable page', () => {
```

Comment:
So, my thinking was that the management of these tabs has been fully moved over into the new component. I think there is even potential to remove the "Top Nav" component pattern entirely and just use the new bar which is managing the display ` <HorizontalNavBarWithOverflowMenu />`
It seemed like that would be better addressed as a follow up issue, since it would require some refactoring and making sure there are no regressions across many plugins.
I will double check the deleted file and see if there is any test that I have lost in this process though, that should still be present.

Reply:
Ah right, yeah that makes sense, so I think it's fine, thank you